### DISCUSSION TITLE
View connections

### DISCUSSION THREAD

**Discussion by keibol** [mattieserver/netbox-topology-views#205] [Q&A]
Hello
What is the correct syntax for viewing connections?

'enable_circuit_terminations': 'True',

I edited this way but it didn't work

![image](https://user-images.githubusercontent.com/95502707/165966266-a783415c-98b8-4c34-801c-133e55c382cb.png)

**Comment by ekrichbaum** [comment]
```
[root@netbox netbox]# grep enable_circuit configuration.py 
                'enable_circuit_terminations': False
```
so in your case just True.

**Comment by keibol** [comment]
Sorry, I think I expressed myself badly.

What I need is for these connections to appear in the plugin.

![image](https://user-images.githubusercontent.com/95502707/166104357-f1a115ec-9195-44b4-b8c2-7893015e2ff2.png)

Is this feature available?

**Comment by joaolucasmacedo** [comment]
Maybe you are talking about the same issue #45
